src/vivarium_census_prl_synth_pop/components/observers.py

```diff
@@ -308,20 +311,13 @@ def name(self):
     def output_filename(self):
         return f"wic.hdf"
 
-    def setup(self, builder: Builder):
-        super().setup(builder)
-        self.time_step = builder.configuration.time.step_size  # in days
-        assert (
-            1 <= self.time_step <= 30
-        ), "WICObserver requires model specification configuration with 1 <= time.step_size <= 30"
-        self.randomness = builder.randomness.get_stream(self.name)
-
-    def get_population_view(self, builder) -> PopulationView:
-        """Get the population view to be used for observations"""
-        return builder.population.get_view(columns=metadata.WIC_OBSERVER_COLUMNS_USED)
+    @property
+    def input_columns(self):
+        return self.DEFAULT_INPUT_COLUMNS + self.ADDITIONAL_INPUT_COLUMNS
 
-    def get_responses(self) -> pd.DataFrame:
-        self.response_columns = [
+    @property
```

Comment:
I just realized we spent all that time talking about input columns, but we really should do the same thing for output columns.